### PR TITLE
feat: add cell module

### PR DESCRIPTION
**Pull request by classicalliu**

### CODE REVIEW

File: packages/neuron-ui/src/containers/Sidebar/index.tsx [hunk 1]
```diff
@@ -60,6 +61,10 @@ const walletMenuItems = [
     name: 'History',
     icon: IconHistory,
   },
+  {
```
Comment (ashchan) [inline packages/neuron-ui/src/containers/Sidebar/index.tsx]: Think twice if an end user should have to know what cells are.

File: packages/neuron-ui/src/locales/zh.ts [hunk 2]
```diff
@@ -9,6 +9,7 @@ export default {
     Send: '转账',
     Receive: '收款',
     History: '交易历史',
+    UnspentCells: '未使用Cells',
```
Comment (ashchan) [inline packages/neuron-ui/src/locales/zh.ts]: Like commented above, I don't thin the wallet interface should expose cells data. For a wallet user, she/he only cares about that she/he could spend amount <= balance. No need to expose cells concept to user.

File: packages/neuron-wallet/src/channel/index.ts [hunk 3]
```diff
@@ -127,7 +128,7 @@ const listenToChannel = () => {
     setTimeout(() => {
       e.sender.send(Channel.GetCellsByTypeHash, {
         status: 1,
-        result: [cell],
+        result: ['cells'],
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/channel/index.ts]: Why not return response of `getUnspentCells`

Comment (classicalliu) [reply packages/neuron-wallet/src/channel/index.ts]: Not used now, not important

**Pull request merged**